code/modules/paperwork/pen.dm

```diff
@@ -120,6 +148,13 @@
 						"Command Blue" = "pen-fountain-cb"
 						)
 	embedding = list("embed_chance" = 75)
+	dart_insert_casing_icon_state = "overlay_fountainpen_gold"
+	dart_insert_projectile_icon_state = "overlay_fountainpen_gold_proj"
+	var/list/overlay_reskin = list("Oak" = "overlay_fountainpen_gold",
+								"Gold" = "overlay_fountainpen_gold",
+								"Rosewood" = "overlay_fountainpen_gold",
+								"Black and Silver" = "overlay_fountainpen",
+								"Command Blue" = "overlay_fountainpen_gold")
```

Comment:
```suggestion
	var/list/overlay_reskin = list(
		"Oak" = "overlay_fountainpen_gold",
		"Gold" = "overlay_fountainpen_gold",
		"Rosewood" = "overlay_fountainpen_gold",
		"Black and Silver" = "overlay_fountainpen",
		"Command Blue" = "overlay_fountainpen_gold",
	)
```

this is the proper way to multiline. also you should alphabetize this list unless there's a reason not to

Reply:
It was in the same order as was defined in the `unique_reskin` list. If you want me to alphabetize `overlay_reskin`, then `unique_reskin` should probably also be alphabetized.